README.md

```diff
@@ -56,11 +68,15 @@ $ gcloud config set project $PROJECT_ID
 # Get App Engine component
 $ gcloud components update app
 
+# Setup Managed VMs (select option [4] to install all runtimes)
+gcloud preview app setup-managed-vms
+
 # Check that Docker is running
 $ boot2docker up
 
-# Download the Node.js Docker image
+# Download the Node.js Docker image and Google Docker registry
 $ docker pull google/nodejs-runtime
+$ docker pull google/docker-registry
```

Comment:
actually now that google/docker-registry is getting decommissioned, we should prob stick to `setup-managed-vms` to be future proof.

Sorry for the laggy back and forth.